sdks/python/apache_beam/transforms/core.py

```diff
@@ -1620,6 +1622,11 @@ def with_exception_handling(
           defaults to the windowing of the input.
       timeout: If the element has not finished processing in timeout seconds,
           raise a TimeoutError.  Defaults to None, meaning no time limit.
+      on_failure_callback: If an element fails or times out,
+          on_failure_callback will be invoked. It will receive the exception
+          and the element being processed in as args. Be careful with this
```

Comment:
>  It will receive the exception and the element being processed in as args

What will be the exception when the error is a timeout?

Reply:
It will be a `TimeoutError`, updated the docstring - thanks